fe/src/project/AspNetCore/shared/components/Header/Header.js

```diff
@@ -12,10 +12,10 @@ import {ShoppingCart} from '@material-ui/icons';
 import AuthenticationHelper from "../../../../../shared/helpers/globals/AuthenticationHelper";
 
 function Header(props) {
-
+    const [state, dispatch] = useContext(Context);
     const [searchTerm, setSearchTerm] = useState(props.searchTerm ? props.searchTerm : "");
     const [suggestions, setSuggestions] = useState([]);
-    const [, dispatch] = useContext(Context);
+    const [totalBasketItems, setTotalBasketItems] = useState(0);
```

Comment:
**suggestion:** change to:

const [totalBasketItems, setTotalBasketItems] = useState(props.totalBasketItems ? props.totalBasketItems : 0);